yb-voyager/cmd/importDataFileTaskPicker.go

```diff
@@ -45,39 +49,57 @@ func NewSequentialTaskPicker(tasks []*ImportFileTask, state *ImportDataState) (*
 		switch taskStatus {
 		case FILE_IMPORT_COMPLETED:
 			doneTasks = append(doneTasks, task)
-		case FILE_IMPORT_NOT_STARTED, FILE_IMPORT_IN_PROGRESS:
+		case FILE_IMPORT_NOT_STARTED:
 			pendingTasks = append(pendingTasks, task)
+		case FILE_IMPORT_IN_PROGRESS:
+			if inProgressTask != nil {
+				return nil, fmt.Errorf("multiple tasks are in progress. task1: %v, task2: %v", inProgressTask, task)
+			}
```

Comment:
what can be the case for this state(2 tasks in progress for sequential picker)?

Reply:
It should never be the case, that's why returning an error (just as a guardrail).